official/projects/fffner/README.md

```diff
@@ -0,0 +1,18 @@
+# FFF-NER tf ver.
```

Comment:
Could you update the README.md using the format as https://github.com/tensorflow/models/tree/master/official/projects/mosaic?